private/tools/java/com/github/bazelbuild/rules_jvm_external/resolver/maven/MavenCoordinates.java

```diff
@@ -0,0 +1,49 @@
+package com.github.bazelbuild.rules_jvm_external.resolver.maven;
+
+import static com.github.bazelbuild.rules_jvm_external.resolver.maven.MavenPackagingMappings.mapPackagingToExtension;
+import static com.github.bazelbuild.rules_jvm_external.resolver.maven.MavenPackagingMappings.mapPackingToClassifier;
+
+import com.github.bazelbuild.rules_jvm_external.Coordinates;
+import org.eclipse.aether.artifact.Artifact;
+
+class MavenCoordinates {
+
+  public static Coordinates asCoordinates(Artifact artifact) {
+    // Please excuse the sleight-of-hand here. Turns out that we need to do some finagaling of the
```

Comment:
Do you mind commenting on what the sleight-of-hand is?  Is it just fixing the classifier?